Hung beam.smp sitting at 100% CPU

## Description

```irc
13:56 <+davisp> 1) I've seen some test cases sit idle for 40-50m+. SSH'ing to the node
                holding that job shows beam.smp spinning at ~100% CPU on the node.
                Unfortunately using `kill -SIGUSR1 $pid` to try and generate a crash dump
                does not work.
```

## Steps to Reproduce

Unsure so far. Perhaps an EUnit soak test would work.

## Expected Behaviour

beam should not hang.

## Your Environment

This has occured on IBM Cloud (x86_64, docker) workers as well as our FreeBSD (12.1, iocage, no docker) workers during various builds.

## Additional Context

Unclear if this is specific to Jenkins, to the version of Erlang we're using, or if this is genuinely a release blocking issue.

```
[2020-01-23T15:08:02.163Z]   [done in 0.468 s]

[2020-01-23T15:08:02.163Z] module 'mem3_reshard_test'

[2020-01-23T15:08:02.163Z]   mem3 shard split db tests

[2020-01-23T15:08:03.263Z]     mem3_reshard_test:86: split_one_shard...[0.459 s] ok

[2020-01-23T15:08:03.963Z]     mem3_reshard_test:146: update_docs_before_topoff1...[0.466 s] ok

[2020-01-23T15:44:16.999Z] Sending interrupt signal to process

[2020-01-23T15:44:17.658Z] Sending interrupt signal to process

[2020-01-23T15:44:26.285Z] Terminated

[2020-01-23T15:44:26.285Z] make[1]: *** [Makefile:175: eunit] Terminated

[2020-01-23T15:44:26.285Z] make: *** [Makefile:153: check] Terminated

[2020-01-23T15:44:26.285Z]     undefined

[2020-01-23T15:44:26.285Z]     *** context setup failed ***

[2020-01-23T15:44:26.285Z] **in function mem3_reshard_test:with_proc/3 (test/eunit/mem3_reshard_test.erl, line 685)

[2020-01-23T15:44:26.285Z] in call from mem3_reshard_test:setup/0 (test/eunit/mem3_reshard_test.erl, line 35)

[2020-01-23T15:44:26.285Z] **error:{noproc,{gen_server,call,[mem3_nodes,get_nodelist]}}

[2020-01-23T15:44:26.285Z] 

[2020-01-23T15:44:26.285Z] 

[2020-01-23T15:44:26.285Z]   undefined

[2020-01-23T15:44:26.285Z]   *** context setup failed ***

[2020-01-23T15:44:26.285Z] **in function mem3_reshard_test:with_proc/3 (test/eunit/mem3_reshard_test.erl, line 685)

[2020-01-23T15:44:26.285Z] in call from mem3_reshard_test:setup/0 (test/eunit/mem3_reshard_test.erl, line 35)

[2020-01-23T15:44:26.285Z] **error:{noproc,{gen_server,call,[mem3_nodes,get_nodelist]}}
```

Another case of a stuck scheduler I think. It looks like one test finished, then the next one was being set up. In the setup code we call `mem3_nodes:get_node_list()` that call however never returned and everything was stuck waiting for it.

I clicked either stop or restart make check and it then received a bunch of termination signals and it seems to have killed mem3_nodes gen_server and all the stuck waiting calls returned with noproc. And then the whole thing got torn down.

--- a/src/couch/src/couch_util.erl
+++ b/src/couch/src/couch_util.erl
@@ -692,7 +692,7 @@ set_mqd_off_heap(Module) ->
 
 
 set_process_priority(Module, Level) ->
-    case config:get_boolean("process_priority", atom_to_list(Module), true) of
+    case config:get_boolean("process_priority", atom_to_list(Module), false) of
         true ->
             process_flag(priority, Level),
             ok;

